core/src/main/java/jenkins/util/AntClassLoader.java

```diff
@@ -924,22 +926,25 @@ public URL getResource(final String name) {
     }
 
     /**
-     * Finds a matching file by iterating pathComponents.
+     * Finds a matching file by iterating through path components.
      *
-     * @param pathComponents Path to a folder, split into the individual folder names
      * @param name File to find
-     * @return Url to found object
+     * @return A {@code URL} object for reading the resource, or {@code null} if the
+     *     resource could not be found
      */
-    @CheckForNull
-    protected URL getUrl(Iterable<File> pathComponents, String name) {
+    private URL getUrl(String name) {
         URL url = null;
+
+        // try and load from this loader if the parent either didn't find
+        // it or wasn't consulted.
         for (final File pathComponent : pathComponents) {
             url = getResourceURL(pathComponent, name);
             if (url != null) {
                 log("Resource " + name + " loaded from ant loader", Project.MSG_DEBUG);
                 break;
             }
         }
+
         return url;
     }
```

Comment:
I cleaned up and upstreamed this change in apache/ant#150. This just adjusts our local copy to match the final version that was accepted upstream.